src/views/Konnekt.vue

```diff
@@ -1,24 +1,30 @@
 <template>
-  <div class="konnekt-background">
+  <div :class="{ 'konnekt-background': isHome }">
     <Toolbar :title="this.$options.name" />
-    <Home />
+    <v-flex>
+      <router-view />
+    </v-flex>
   </div>
 </template>
 
 <script>
-import Home from "../components/KonnektHome";
 import Toolbar from "../components/Toolbar";
 
 export default {
   name: "Konnekt",
   components: {
-    Toolbar,
-    Home
+    Toolbar
+  },
+  computed: {
```

Comment:
Computed property doesn't work properly. I tried another way:
```js
  data: () => ({ isHome: true }),
  mounted() {
    this.updateBackground();
  },
  methods: {
    updateBackground() { this.isHome = this.$router.currentRoute.path.toString() === "/konnekt"; }
  },
  watch: {
    `$route` (to, from) {
      this.updateBackground();
    }
  }
```

Reply:
okay Lets use this then.

Reply:
done

Reply:
there's a warning for watch property it says that the parameters `to` and `from` are not used